apps/nexttrainsdc/manifest.yaml

```diff
@@ -0,0 +1,8 @@
+---
+id: next-trains-dc
+name: Next Trains DC
+summary: Shows next trains time
```

Comment:
Could you update the summary and description so that users understand it's train times for a specific stop in the network, and not all of DC?